src/lib.rs

```diff
@@ -365,4 +365,13 @@ impl JsTaggedBase64 {
     pub fn set_value(&mut self, value: &[u8]) {
         self.tb64.set_value(value);
     }
+
+    /// Formats the JsTaggedBase64 instance as a URL-safe string.
+    //
+    // Note: this method is included for WASM bindings, since the trait methods from Display don't
+    // get compiled to WASM.
+    #[allow(clippy::inherent_to_string_shadow_display)]
```

Comment:
Oh, huh. We have https://github.com/SpectrumXYZ/tagged-base64/blob/0159640927e5e1bfeac431f121362c70f0ca7e94/src/lib.rs#L121 for the TaggedBase64 type, it's odd that we don't have the same for the JS Wrapper.

Is a method, as opposed to a standalone function, preferable here? Overloaded name aside?

Reply:
Incidentally... there's an outstanding issue around this, not a lot of activity. https://github.com/rustwasm/wasm-bindgen/issues/2073